src/frequenz/sdk/actor/_channel_registry.py

```diff
@@ -49,3 +49,15 @@ def new_receiver(self, key: str) -> Receiver[Any]:
         if key not in self._channels:
             self._channels[key] = Broadcast(f"{self._name}-{key}")
         return self._channels[key].new_receiver()
+
+    async def _close_channel(self, key: str) -> None:
```

Comment:
@sahas-subramanian-frequenz suggested that so that it will not be readily used as it invalidates all the senders of a channel when run.

The idea was that it should always be exceptional if/when it is used.

Reply:
OK, let's leave it like this as this needs further alignment. :+1: 

Probably the whole channel registry should be hidden to the users in the future anyway...